lib/a.json

```diff
@@ -39,6 +39,7 @@
   "one",
   "one/*",
   "one-*",
+  "one:*",
```

Comment:
"1:1" is internally transformed to "one:1", while "1:00" is transferred to "one:00". This pattern will match both.